src/test/java/core/basesyntax/servicesimpl/ResultMessageImplTest.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.servicesimpl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.FruitDaoImpl;
+import core.basesyntax.db.DataBase;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.services.ResultMessage;
+import core.basesyntax.strategy.ElementDoesNotExist;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ResultMessageImplTest {
+    private ResultMessage resultMessage;
+    private Fruit lemon;

```

Comment:
you use this field only in one test method, so make it local and use it only in this test method

Reply:
✅